python/ray/_private/runtime_env.py

```diff
@@ -194,9 +194,11 @@ def __init__(self, runtime_env_json: dict):
         # Used for testing wheels that have not yet been merged into master.
         # If this is set to True, then we do not inject Ray into the conda
         # or pip dependencies.
-        if "_skip_inject_ray" in runtime_env_json:
-            self._dict["_skip_inject_ray"] = runtime_env_json[
-                "_skip_inject_ray"]
+        if os.environ["RAY_RUNTIME_ENV_LOCAL_DEV_MODE"]:
```

Comment:
Should we leave `_skip_inject_ray` in as well as an "escape hatch" for unexpected situations in the future?  Maybe if someone needs to inject some specific wheel link or something--I don't have a concrete example of when it would be needed, so I don't know.  I know release versions and master commits are covered by the existing injection, and local builds from source are covered by your contribution, but I'm not sure if there are other cases I haven't thought of.

Reply:
I saw `os.environ["RAY_RUNTIME_ENV_LOCAL_DEV_MODE"]` is giving a KeyError in some tests, maybe we should use `if "RAY_RUNTIME_ENV_LOCAL_DEV_MODE" in os.environ:`

Reply:
i think wheel link should still work?

Reply:
Oh you're right